<!-- Launch file to move on a preprogrammed trajectory -->

<launch>
	<group ns="ias"> <group ns="robot"> 
	
		<arg name="arm_name" default="arm_1"/>
		<arg name="rate" default="100"/>
	
		<group ns="$(arg arm_name)">  	
  	
			<node name="gesture_server" pkg="irob_autosurg" 
			type="gesture_main" output="screen">
  		
					<param name="arm_name" 
						type="string" value="$(arg arm_name)" />
					<param name="rate" type="double" value="$(arg rate)" />
					
					<remap from="position_cartesian_current_in"
							 to="position_cartesian_current" />
	</node>
  	</group>
  	</group> </group>
</launch>
